src/graph/GoExecutor.cpp

```diff
@@ -990,13 +1020,12 @@ bool GoExecutor::processFinalResult(RpcResponse &rpcResp, Callback cb) const {
                                         "Get edge type for `%s' failed in getters.",
                                         edgeName.c_str());
                             }
-                            if (type != edgeType) {
+                            if (type != std::abs(edgeType)) {
                                 return 0L;
                             }
                         }
-                        return isReversely() ? srcId : dstId;
+                        return dstId;
```

Comment:
@whitewum Please note the changes here.

Reply:
can anybody tell me:

E.g.,
```
Match (n)-[xx]->(m)
Return id(m)
```
is 
```
GO FROM n OVER xx YIELD xx._dst
```

```
Match (m)<-[xx]-(n)
Return id(n)
```
is it 
```
GO FROM m REVERSELY OVER xx YIELD xx._dst or xx._src
```

I'm confused several times, and can't get it anymore.

Reply:
e.g.2
GO FROM m REVERSELY OVER xx YIELD xx._dst 

e.g.3
GO FROM n BIDIRECT OVER xx YIELD xx._dst


You can think the "_src" represents myself,   "_dst" means the other point.